api/v1beta1/flinkcluster_types.go

```diff
@@ -383,6 +384,9 @@ type FlinkClusterSpec struct {
 
 	// Config for GCP.
 	GCPConfig *GCPConfig `json:"gcpConfig,omitempty"`
+
+	// Revision history limit to keep.
```

Comment:
The maximum number of revision history to keep.

Reply:
Done.